util.py

```diff
@@ -160,3 +160,18 @@ def length_check(banner_message: str) -> Tuple[str, bool]:
     elif len(banner_message) > max_length:
         return "Banner message too long.", False
     return None, True
+
+
+def get_theme_directories(theme_path: str) -> List[str]:
+    """
+    Function to retrieve theme directory names in the specified path, sorted alphabetically.
```

Comment:
I was thinking, is there a reason to sort them? We could return them as a set or an unsorted list.

Reply:
Yes, I didn't sort them initially and it looked messy as a long dropdown list for selection. In addition, when new themes added, sorted list gives some intuition to find the added theme easily.

Reply:
Ah I see